src/data/valid/DataObject-exhaustive.yaml

```diff
@@ -10,4 +10,4 @@ md5_checksum: "22afa3d49b73eaec2e9787a6b88fbdc3"
 name: Crispr Terms
 type: "nmdc:DataObject"
 url: http://example.com
-was_generated_by: "nmdc:invalid_id"
+was_generated_by: "nmdc:act-99-123456.1"
```

Comment:
Update to WorkflowExecutionActivity subclass instance. Activity is abstract so won't mint an identifier.